Src/ElasticScale.Client/ShardManagement/ShardMap/ListShardMap.cs

```diff
@@ -100,9 +100,34 @@ public SqlConnection OpenConnectionForKey(TKey key, string connectionString, Sql
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1061:DoNotHideBaseClassMethods")]
         public SqlConnection OpenConnectionForKey(TKey key, string connectionString, ConnectionOptions options)
         {
-            return this.OpenConnectionForKey(key, connectionString, null, options);
+            return this.OpenConnectionForKey(key, connectionString, null, null, options);
         }
 
+        /// <summary>
+        /// Opens a regular <see cref="SqlConnection"/> to the shard 
+        /// to which the specified key value is mapped.
+        /// </summary>
+        /// <param name="key">Input key value.</param>
+        /// <param name="connectionString">
+        /// Connection string with credential information such as SQL Server credentials or Integrated Security settings. 
+        /// The hostname of the server and the database name for the shard are obtained from the lookup operation for key.
+        /// </param>
+        /// <param name="accessToken">Secure SQL Access token</param>
```

Comment:
Inconsistent param documentation here

Reply:
Please make all param documentation for accessToken identical.

Reply:
done!